docs/index.asciidoc

```diff
@@ -129,7 +129,7 @@ filter {
 [id="plugins-{type}s-{plugin}-ttl"]
 ===== `ttl`
 
-For usages of this plugin that persist data to memcached (e.g., <<#plugins-{type}s-{plugin}-set,`set`>>), the time-to-live in seconds
+For usages of this plugin that persist data to memcached (e.g., <<{version}-plugins-{type}s-{plugin}-set,`set`>>), the time-to-live in seconds
```

Comment:
Do you need the `{version}` in the [id=  ] section too? `[id="plugins-{type}s-{plugin}-set"]`

Reply:
@guyboertje   Actually, I don't need it anywhere in the source file.  Great catch! Thank you.